cmd/erasure.go

```diff
@@ -182,7 +172,7 @@ func getDisksInfo(disks []StorageAPI, endpoints []string) (disksInfo []madmin.Di
 	// Wait for the routines.
 	for i, diskInfoErr := range errs {
 		ep := disksInfo[i].Endpoint
-		if diskInfoErr != nil {
+		if diskInfoErr != nil && !errors.Is(diskInfoErr, errUnformattedDisk) {
```

Comment:
An unformatted disk is not considered as an offline drive - let me know if this acceptable for you in vsphere plugin @dvaldivia @cesnietor